### PR TITLE
Generate RPC implementation

### PR DESCRIPTION
**Pull request by erizocosmico**
Generate implementation of protobuf services in Go to use with gRPC.

### Missing stuff

* Command to run this (probably will do it in another PR after this is merged)
* Full integration test (requires implementation on the scanner, so it can't be done yet)

### CODE REVIEW

**Comment by erizocosmico** [comment]
Reminder: no variadicity implemented

**Comment by erizocosmico** [comment]
**NOTE** not ready to be reviewed yet

**Comment by erizocosmico** [comment]
@Serabe @dpordomingo can be reviewed again

File: protobuf/protobuf.go [hunk 1]
```diff
@@ -212,3 +230,19 @@ type EnumValue struct {
...
 }
+
+// RPC is a single exposed RPC method in the RPC service.
+type RPC struct {
+	Name string
+	// Recv is the name of the receiver Go type. Empty if it's not a method.
+	Recv string
+	// Method is the name of the Go method or function.
+	Method string
+	// HasError repors whether the Go function returns an error.
```
Comment (Serabe) [inline protobuf/protobuf.go]: s/repors/reports/

File: protobuf/gen_test.go [hunk 2]
```diff
@@ -137,6 +137,34 @@ func (s *GenSuite) TestWriteMessage() {
...
+		Output: NewNamed("foo.bar", "DoFooResponse"),
+	},
+	{
+		Name:   "DoBar",
+		Input:  NewNamed("foo.bar", "DoBarRequest"),
+		Output: NewNamed("foo.bar", "DoBarResponse"),
+	},
+}
+
+const expectedService = `service BarService {
```
Comment (erizocosmico) [inline protobuf/gen_test.go]: Not at this exact point, but the imports are being tracked

File: protobuf/protobuf.go [hunk 3]
```diff
@@ -41,6 +43,13 @@ func (p *Package) isImported(file string) bool {
 	return false
 }
 
+// ServiceName returns the RPC service name of the package.
```
Comment (erizocosmico) [inline protobuf/protobuf.go]: Yep, true

File: protobuf/protobuf.go (hunk 1 above)
Comment (erizocosmico) [inline protobuf/protobuf.go]: i thought i fixed this somewhere else, but apparently not

**Comment by erizocosmico** [comment]
Fixed @Serabe

**Pull request merged**